docs/source-pytorch/common/evaluation_basic.rst

```diff
@@ -124,7 +124,8 @@ To run the validation loop, pass in the validation set to **.fit**
 
    train_loader = DataLoader(train_set)
    valid_loader = DataLoader(valid_set)
+   model = LitAutoEncoder(...)
 
    # train with both splits
-   trainer = Trainer()
+   trainer = pl.Trainer()
```

Comment:
Hmm, we use `Trainer()` or `L.Trainer()` everywhere else in the docs. I think it doesn't make much sense to add the `pl.` if we're not importing it anywhere.

Reply:
Yes, maybe we should explicitly import it or remove it in both places. Right now it's just inconsistent.
<img width="899" alt="Screenshot 2023-10-23 at 6 55 00 PM" src="https://github.com/Lightning-AI/lightning/assets/5618407/3e025b83-5738-4825-909b-77dd18602e54">

Reply:
> Hmm, we use Trainer() or L.Trainer() everywhere else in the docs

In the Level 1 Beginner docs it's `pl`: https://lightning.ai/docs/pytorch/stable/model/train_model_basic.html
<img width="878" alt="Screenshot 2023-10-23 at 6 56 50 PM" src="https://github.com/Lightning-AI/lightning/assets/5618407/72cd35a7-7cc8-426c-b648-e026011c04bc">

Reply:
lets drop it in both places :)

Reply:
I'll do the rest here: #18869
We can still get the small fix from @rasbt in like this IMO.

Reply:
3 and 4 are the goal for our docs. 4 depends on the context, sometimes things need to be imported that aren't available from the top lightning module, so you need to go deeper.

Anything with pytorch_lightning (1 and 2) definitely not, this should not be found in any of our recent docs :)